src/templates/lesson.jsx

```diff
@@ -105,52 +105,38 @@ const ToCContainer = styled.div`
 /* eslint no-undef: "off"*/
 export const pageQuery = graphql`
   query LessonBySlug($slug: String!) {
-  postBySlug: markdownRemark(fields: { slug: { eq: $slug } }) {
-    html
-    timeToRead
-    excerpt
-    frontmatter {
-      title
-      cover
-      date
-      category
-      tags
+    postBySlug: markdownRemark(fields: { slug: { eq: $slug } }) {
```

Comment:
The query is now very general and allows for any number of chapters and subchapters in the `table_of_contens.json`. Just one place where you have to add a new chapter, not two which have to be in sync (which was driving me crazy).